pkg/crc/validation/validation.go

```diff
@@ -85,6 +75,21 @@ func ValidateBundle(bundlePath string, preset crcpreset.Preset) error {
 	return nil
 }
 
+func bundleMismatchWarning(userProvidedBundle string, preset crcpreset.Preset) {
+	userProvidedBundle = bundle.GetBundleNameWithExtension(userProvidedBundle)
```

Comment:
I'd prefer if this `GetBundleNameWithExtension` change was done in the commit introducing GetBundleNameWithExtension, preferrably with an explanation why it's needed. Hard to understand why it's needed now when it's done silently in a commit which is supposed to be only code movement.

Reply:
@cfergeau you mean squashing those 2 commits or update the current commit by saying it is used in following commit message?

Reply:
```
commit 494a512d4223f96a99619996dec3a834f2a1efe4
Author: Praveen Kumar <kumarpraveen.nitdgp@gmail.com>
Date:   Thu Sep 1 16:12:12 2022 +0530

    Bundle: Add helper function to get bundle name with extension
    
    This new method is used in pkg/crc/validation/validation.go in order to
    solve XXX

diff --git a/pkg/crc/machine/bundle/metadata.go b/pkg/crc/machine/bundle/metadata.go
index 2fd8f1b6..754fb991 100644
--- a/pkg/crc/machine/bundle/metadata.go
+++ b/pkg/crc/machine/bundle/metadata.go
@@ -237,6 +237,13 @@ func GetBundleNameWithoutExtension(bundleName string) string {
 	return strings.TrimSuffix(bundleName, bundleExtension)
 }
 
+func GetBundleNameWithExtension(bundleName string) string {
+	if strings.HasSuffix(bundleName, bundleExtension) {
+		return bundleName
+	}
+	return fmt.Sprintf("%s.%s", bundleName, bundleExtension)
+}
+
 func GetCustomBundleName(bundleFilename string) string {
 	re := regexp.MustCompile(`(?:_[0-9]+)*.crcbundle$`)
 	baseName := re.ReplaceAllLiteralString(bundleFilename, "")
diff --git a/pkg/crc/validation/validation.go b/pkg/crc/validation/validation.go
index 8993e975..4139ccae 100644
--- a/pkg/crc/validation/validation.go
+++ b/pkg/crc/validation/validation.go
@@ -61,6 +61,7 @@ func ValidateBundlePath(bundlePath string, preset crcpreset.Preset) error {
 	}
 
 	userProvidedBundle := filepath.Base(bundlePath)
+	userProvidedBundle = bundle.GetBundleNameWithExtension(userProvidedBundle)
 	if userProvidedBundle != constants.GetDefaultBundle(preset) {
 		// Should append underscore (_) here, as we don't want crc_libvirt_4.7.15.crcbundle
 		// to be detected as a custom bundle for crc_libvirt_4.7.1.crcbundle
```

Reply:
> Hard to understand why it's needed now when it's done silently in a commit which is supposed to be only code movement.

Unresolving this comment... :( The previous commit has an explanation now, not fully clear if this added call will change behaviour right away or not. I still feel this belongs in the previous commit, with a clear statement whether `userProvidedBundle = bundle.GetBundleNameWithExtension(userProvidedBundle)` changes the current behaviour, or not.